### ISSUE TITLE
uTox version : 0.9.4 - feedback

### ISSUE THREAD

**Issue by darkman088** [GrayHatter/uTox#390]
Dear uTox developers,

I am writing here BIG WORDS and BIG THANKS!!! Version 0.9.4 of uTox is again MUCH BETTER,
MUCH SMARTER, I see huge improvements!
The highest value I find in the improved quality during voice calls, which is soooo much better than before !! Now it is REALLY USABLE! It's also great, that the balloon for incoming messages is back, 
and it's really cool, that uTox produces a light sound, when contacts go online and offline! Very good job!!! I also like the tiny contact list very much !!

I also have to report a little bug, which probably missed your attention - ordering the contact list is not persistent over uTox restarts. It is, of course, still possible to reorder contacts in the order I would like to see them, though when I quit uTox and start it again, they are once more displayed as
before, as if I didn't order them previously. I would be really delighted, if you could fix that and also - what is very important - I still find less and less things to report !!! uTox is getting really mature!

THANK you so much for your huge efforts! Keep up the good work !!! THANKS !!!

**Comment by fcore117** [comment]
yes i too like that sounds are modern and soft like that do not shock me when speakers are in high level.
